packages/gatsby-cli/src/create-cli.js

```diff
@@ -266,7 +266,7 @@ function buildLocalCommands(cli, isLocalSite) {
 
   cli.command({
     command: `repl`,
-    desc: `Get a node repl with context of Gatsby environment, see (add docs link here)`,
+    desc: `Get a node repl with context of Gatsby environment, see (https://www.gatsbyjs.org/docs/gatsby-repl/)`,
```

Comment:
```suggestion
    desc: `Get a node repl with context of Gatsby environment, see (/docs/gatsby-repl/)`,
```

This can just be a relative link 🙂

Reply:
Hey @gillkyle, thanks for the review and the corrections! I've never been the best speller 😂.

I'm pushing up a new commit now with those changes. I'll get the WIP removed and add more documentation here over the next couple of days.

Thanks again!